visualisation/aequatus/static/aequatus.css

```diff
@@ -0,0 +1,137 @@
+.circleBase {
+    border-radius: 50%;
+    behavior: url(PIE.htc); /* remove if you don't care about IE8 */
+}
+
+.type2 {
+    width: 20px;
+    height: 20px;
+    background: #ccc;
+    border: 1px solid #4682b4;
+}
+
+#info_div{
+    display: none; 
+    background: none repeat scroll 0% 0% peru; 
+    padding: 10px; 
+    height: 248px; 
+    text-align: center; 
+    font-size: 20px;
+}
+
+#settings_div{
+    padding: 10px; 
+    height: 248px; 
+    background: none repeat scroll 0% 0% darkcyan; 
+    font-size: large;
+}
+
+#control_panel{
+    top: 100px;
+    position: fixed; 
+    height: auto; 
+    left: -300px; 
+    z-index: 2999
+}
+
+#control_panel_handle{
+    padding: 5px; 
+    text-align: center; 
+    cursor: move; 
+    background: none repeat scroll 0% 0% slategrey; 
+    color: white;
+}
+
+#settings_div_handle{
+    padding: 5px; 
+    text-align: center; 
+    background: none repeat scroll 0% 0% darkcyan;
+}
+
+#info_panel_handle{
+    padding: 5px; 
+    text-align: center; 
+    background: none repeat scroll 0% 0% peru;
+}
+
+#openclose_handle{
+    padding: 5px; 
+    text-align: center;  
+    background: none repeat scroll 0% 0% gray;
+}
+
+#gene_tree{
+    overflow: visible;   
+    position: relative; 
+    top: 50px; 
+    left: 50px;
+    width: 100%;
+}
+
+#popup{
+    width:200px; 
+    height:130px;
+    display: none;
+}
+
+#stable_id_header{
+    color: white; 
+    padding: 2px; 
+    width: 100%;
+}
+
+#makemetop_button{
+    float: right
+}
+
+#popup_header{
+    overflow: hidden; 
+    left: 0px; 
+    top: 0px; 
+    position: relative;
+}
+
+#popup_body{
+    position: relative; 
+    padding: 5px;
+}
+
+#ruler {
+    font-family: Courier;
+    font-size: 13px;
+    visibility: hidden;
+    white-space: nowrap;
+    width: auto;
+}
+
+.bubbleright {
+     background-color: white;
+     background-image: -moz-linear-gradient(center top , rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0));
```

Comment:
`background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0));`